resources/views/PhpReboot/Magazine/monthPage.blade.php

```diff
@@ -1,6 +1,6 @@
 @extends('layouts.Admin-LTE-2-3-4.master')
 
-@section('title', 'Meetup')
+@section('title', $magazine->name . 'magazine')
```

Comment:
These changes are not related to login form.

This probably happened due to wrong branch creation/merges.